tensorflow_lite_support/ios/task/processor/sources/TFLSegmentationResult.h

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** Holds a label associated with an RGB color, for display purposes. */
+@interface TFLColoredLabel : NSObject
+
+/** The RGB color components for the label, in the [0, 255] range. */
+@property(nonatomic, assign) NSUInteger r;
+@property(nonatomic, assign) NSUInteger g;
+@property(nonatomic, assign) NSUInteger b;
+
+/** The class name, as provided in the label map packed in the TFLite Model
+ * Metadata.
+ */
+@property(nonatomic, copy) NSString *label;
+
+/** The display name, as provided in the label map (if available) packed in
+ * the TFLite Model Metadata. See `display_names_locale` field in
+ * ImageSegmenterOptions.
+ */
+@property(nonatomic, copy) NSString *displayName;
+
+@end
+
+/** Encapsulates a resulting segmentation mask and associated metadata. */
+@interface TFLSegmentation : NSObject
+
+/**
+ * The width of the mask. This is an intrinsic parameter of the model being
+ * used, and does not depend on the input image dimensions.
+ */
+@property(nonatomic, assign) NSUInteger width;
```

Comment:
Let's change NSInteger so that it can be used as parameter for for loop. It turned out that I can do `for i in 0...width-1` in Swift if it's NSUInteger.